content/2016/09/06/css-grid-layout.rst

```diff
@@ -0,0 +1,183 @@
+
+In the meantime, Flexbox has been a great addition to our CSS Toolbox, giving us a way to lay content out easier than before. Remember when vertical centering used to be a huge ordeal? There are a number of issues that are “Solved by Flexbox” but it is still not a complete grid solution.
+
+If you haven’t used flexbox yet, I highly suggest studying this CodePen demo:
+
+.. raw:: html
+
+   <p data-height="688" data-theme-id="21914" data-slug-hash="adLPwv" data-default-tab="result" data-user="enxaneta" data-embed-version="2" class="codepen">See the Pen <a href="http://codepen.io/enxaneta/pen/adLPwv/">Flexbox playground</a> by Gabi (<a href="http://codepen.io/enxaneta">@enxaneta</a>) on <a href="http://codepen.io">CodePen</a>.</p><script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+
+
+Flexbox is great for single direction elements but lacks the ability to create structure on both x and y axis’. A container with `display: flex;` can either go horizontally or vertically as selected by the flex-direction property. So, we marched on, still in search of creating an easier way to create grids on the web.
+
+CSS Grid Layout to the rescue
+-----------------------------
+
+Well, CSS Grid Layout is in the far distant future. It doesn’t have much browser support at the moment so if you wanted to start using it on a production site you’ll want to wrap it in an ``@supports`` conditional.
+
+.. code:: scss
+
+  .container {
+    display: flex;
+
+    @supports ( display: grid ) {
+  	   display: grid;
+    }
+  }
+
+You can see a few fallbacks in this CodePen demo:
+
+.. raw:: html
+
+  <p data-height="642" data-theme-id="21914" data-slug-hash="vXBvNE" data-default-tab="css,result" data-user="stacy" data-embed-version="2" class="codepen">See the Pen <a href="http://codepen.io/stacy/pen/vXBvNE/">CSS Grid Layout with float and flexbox fallbacks</a> by Stacy (<a href="http://codepen.io/stacy">@stacy</a>) on <a href="http://codepen.io">CodePen</a>.</p><script async src="//assets.codepen.io/assets/embed/ei.js"></script>
+
+
+Grid introduces new vocabulary and CSS properties
+-------------------------------------------------
+
+New grid terminology includes:
+
+.. line-block::
+
+  **Grid line**
+  The lines that create the grid, separating the grid cells.
+
+  **Grid track**
+  The horizontal or vertical space between two grid lines, often spanning multiple grid cells.
+
+  **Grid cell**
+  A single unit of the grid made from the space between four grid lines.
+
+  **Grid area**
+  A group of space between four grid lines, often containing a group of grid cells. Grid areas can be named in CSS.
+
+
+CSS Grid Layout Properties you would use on a grid container:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  - display
```

Comment:
These lists are very technical details, with very little description. I wonder if we should either go into a bit more detail ourselves, or just let the linked content list all the properties? Id be glad for a few more practical examples on where to start, if I want to play with grid layouts today.